scripts/build.sh

```diff
@@ -77,6 +77,7 @@ echo "COPY ${distributions}/*.zip"
 mkdir -p $OUTPUT/plugins
 cp ${distributions}/*.zip ./$OUTPUT/plugins
 
+./gradlew publishToMavenLocal -Dopensearch.version=$VERSION -Dbuild.snapshot=$SNAPSHOT -Dbuild.version_qualifier=$QUALIFIER
```

Comment:
We already have a maven publish github action on pushes. we shouldn't publish in other workflows

Reply:
Are other plugins doing this?